lib/apple/index.js

```diff
@@ -32,14 +46,27 @@ function parseResult(result) {
 		throw error;
 	}
 
-	var receipt = result.receipt;
+	var productId = getReceiptFieldValue(result.receipt, 'product_id');
+	var transactionId = getReceiptFieldValue(result.receipt, 'transaction_id');
+
+	/* jshint camelcase:false */
+    if (result.hasOwnProperty('latest_receipt_info') && result.latest_receipt_info[0]) {
+        latestReceiptInfo = result.latest_receipt_info.sort(function (a, b) {
+            a = parseInt(a.original_purchase_date_ms);
+            b = parseInt(b.original_purchase_date_ms);
```

Comment:
Sorry, this is my bad for not hard-enforcing a style on this project, but would you mind adding a radix argument (`parseInt(str, 10)`) on these two lines?